instance/models/server.py

```diff
@@ -154,8 +155,10 @@ class Server(ValidateModelMixin, TimeStampedModel):
     # State transitions:
     _status_to_building = status.transition(from_states=Status.Pending, to_state=Status.Building)
     _status_to_build_failed = status.transition(from_states=Status.Building, to_state=Status.BuildFailed)
-    _status_to_booting = status.transition(from_states=(Status.Building, Status.Ready), to_state=Status.Booting)
-    _status_to_ready = status.transition(from_states=Status.Booting, to_state=Status.Ready)
+    _status_to_booting = status.transition(
+        from_states=(Status.Building, Status.Ready, Status.Unknown), to_state=Status.Booting
```

Comment:
Initially I thought there wasn't a good way to check for the building state, especially as it's quite ephemeral, but I was able to find the resource I needed. Update coming soon.

Regarding the diagram, I'm fine with adding it if you want.